README.rst

```diff
@@ -294,6 +294,16 @@ by default the data is added to the first found organization, eg::
 
 **Note**: you can follow the `tutorial to migrate database from django-netjsongraph <https://github.com/openwisp/django-netjsongraph/blob/master/README.rst>`_.
 
+create_device_nodes
+^^^^^^^^^^^^^^^^^^^
+
+The ``create_device_nodes`` management command is used to create ``DeviceNode`` objects when
+this module is integrated with ``openwisp_controller``. It can be used as follows:
```

Comment:
I'd say something like:

> This management command can be used to create the initial ``DeviceNode`` relationships when the integration with openwisp-controller (add a link to #integration-with-openwisp-controller-and-openwisp-monitoring) is installed on a pre-existing systems which already has devices in it.

Then, I would add a link to this section at the bottom of #integration-with-openwisp-controller-and-openwisp-monitoring, something like:

> If you're enabling this integration on a pre-existing system, use the create_device_nodes management command (add a link to the management command section) to create the relationships between device and nodes.

**Note**: the part between parenthesis are indications, not text to copy verbatim, ok?